src/TreeKnit.jl

```diff
@@ -6,12 +6,13 @@ using Logging
 using LoggingExtras
 using Parameters
 using Random
+using Combinatorics
```

Comment:
This is used in MultiTreeKnit but not in TreeKnit. Do we need it here?

Reply:
removed :-)